gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/gw7/Packager.java

```diff
@@ -23,16 +25,23 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.stream.StreamSupport'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=137&issues=AWczk9qV0POp3Yiz_Pc5&open=AWczk9qV0POp3Yiz_Pc5)

[ISSUE_KEY]: AWczk9qV0POp3Yiz_Pc5